samwise.js

```diff
@@ -1,5 +1,6 @@
-/**
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.